pkgs/by-name/pa/parca-agent/package.nix

```diff
@@ -0,0 +1,47 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  lib,
+  stdenv,
+}:
+
+buildGoModule rec {
+  pname = "parca-agent";
+  version = "0.35.0";
+
+  src = fetchFromGitHub {
+    owner = "parca-dev";
+    repo = "parca-agent";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-5MoHX47uUtQgszNuu9ImLJPYnaN2NKZKOPa60PMHDL0=";
+    fetchSubmodules = true;
+  };
+
+  proxyVendor = true;
+  vendorHash = "sha256-v2OnCuOI9inZ4WiU/3PbBwK6ZcvZX21MNsLhRWZ6sGY=";
+
+  buildInputs = [
+    stdenv.cc.libc.static
+  ];
+
+  ldflags = [
+    "-X=main.version=${version}"
+    "-X=main.commit=${src.rev}"
+    "-extldflags=-static"
+  ];
+
+  tags = [
+    "osusergo"
+    "netgo"
+  ];
```

Comment:
What is this?

Reply:
Go build tags that need to be specified: https://nixos.org/manual/nixpkgs/stable/#var-go-tags

Background on the `netgo` tag here: https://pkg.go.dev/net#hdr-Name_Resolution, and the `osusergo` tag here: https://pkg.go.dev/os/user#pkg-overview

Specified in the upstream Makefile: https://github.com/parca-dev/parca-agent/blob/d59ef5c9bf21d7b422071c16ae83e01bb5fbcf2b/Makefile#L17